### ISSUE TITLE
Missing `aria-label` attribute from the `BitSeparator` component

### ISSUE THREAD

**Issue by Cyrus-Sushiant** [bitfoundation/bitplatform#8121]
To improve accessibility, we need to add the `aria-label` attribute to the `BitSeparator` component. This will ensure that screen readers can provide a meaningful description of the component for users with disabilities.
